Main.cs

```diff
@@ -1,21 +1,41 @@
-namespace ConsoleApp1;
+﻿namespace HW456;
 
-public class MainClass
+class MainClass
 {
-   static void Main()
-   {
-      Lesson1.Tasks_Lesson1();
-      
-      Lesson2.Task1_CyclesAndConditions();
-      Lesson2.Task2_SortingArray();
-      Lesson2.Task3_Matrix();
-      
-      HW2.Task1_ExistAInArray();
-      HW2.Task2_MinArray();
-      HW2.Task3_Calculator();
-      
-      Lesson3.Task1_InOutputMethods();
-      HW3.Task1_StrDictionary();
-      HW3.Task2_DoublNumbers(); //тут в коде есть ошибка, пока не разобралась где((((
-   }
+    static void Main()
+    {
+        Lesson4.Telephone ph = new Lesson4.Telephone(0,"555666887");
+        Console.WriteLine(ph.PhoneNumberWithoutCode());
+        
+        Func();
+        
+        HW56.Task(); 
+        HW7.Task();
+        
+        Console.ReadLine();
+
+    }
+    
+    public static void Func()
+    {
+        IndividualEntrepreneur individualEntrepreneurClient = new IndividualEntrepreneur()
+        {
+            Id = 65482631,
+            Phone = "+7(499)568945",
+            OrderSum = 512,
+            Date = DateOnly.FromDateTime(DateTime.Now.AddYears(-30)),
+            Name = "Блабла Бла Бла",
+        };

```

Comment:
Обычно используется `var` для уменьшения кода. И скобки можно опускать когда класс инстациируется с указанием полей:
```suggestion        
        var individualEntrepreneurClient = new IndividualEntrepreneur
        {
            Id = 65482631,
            Phone = "+7(499)568945",
            OrderSum = 512,
            Date = DateOnly.FromDateTime(DateTime.Now.AddYears(-30)),
            Name = "Блабла Бла Бла",
        };
```

Reply:
поправила